python-sdk/src/astro/sql/__init__.py

```diff
@@ -4,13 +4,13 @@
 
 from astro.sql.operators.append import AppendOperator, append
 from astro.sql.operators.cleanup import CleanupOperator, cleanup
-from astro.sql.operators.data_validations.ColumnCheckOperator import (  # skipcq: PY-W2000
+from astro.sql.operators.data_validations.check_column import (  # skipcq: PY-W2000
```

Comment:
you can remove `# skipcq` see  https://github.com/astronomer/astro-sdk/pull/1489#discussion_r1071811974

Reply:
Done